plugin/ochttp/http-out-test-cases.json

```diff
@@ -0,0 +1,258 @@
+[
```

Comment:
Usually this type of file is in a `testdata` folder: https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
What's your opinion here - is it better to take a copy from specs or download directly from there? Second approach will fail all incompliant SDKs compilation on any spec modification. But fixes will be implemented faster.

Reply:
Unit tests should never make network calls (except localhost). Better to put it in the repo like you have it here.

Reply:
I made the change